draft/examples/reconciliation-result-batch/valid/minimal.json

```diff
@@ -0,0 +1,28 @@
+{
+  "results": [
+    {
+      "candidates": [
+        {
+          "id": "120333937",
+          "name": "Urbaniak, Regina"
+        },
+        {
+          "id": "1127147390",
```

Comment:
I cannot commit your suggestion because it is identical to the current state, apparently. ("This suggestion is invalid because no changes were made to the code.")